26_Autoexposure/app_resources/common.hlsl

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _AUTOEXPOSURE_COMMON_INCLUDED_
+#define _AUTOEXPOSURE_COMMON_INCLUDED_
+
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+struct AutoexposurePushData
+{
+    float meteringWindowScaleX, meteringWindowScaleY;
+    float meteringWindowOffsetX, meteringWindowOffsetY;
```

Comment:
why not use `float32_t2` instead of splitting for X and Y

Reply:
I didn't realize earlier that `*_t2` types are available C++ side. I'll update these.